phpunit.xml.dist

```diff
@@ -1,16 +1,25 @@
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false"
-         syntaxCheck="false"
-         bootstrap="vendor/autoload.php">
-    <testsuites>
-        <testsuite name="php-tmdb-symfony Test Suite">
-            <directory>./Tests/</directory>
-        </testsuite>
-    </testsuites>
+<?xml version="1.0"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" backupGlobals="false" backupStaticAttributes="false" colors="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" processIsolation="false" stopOnFailure="false" bootstrap="vendor/autoload.php" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+  <coverage processUncoveredFiles="true">
+    <include>
+      <directory>./</directory>
+    </include>
+    <exclude>
+      <directory>./Resources</directory>
+      <directory>./Tests</directory>
+      <directory>./vendor</directory>
+    </exclude>
+  </coverage>
+  <php>
+    <env name="ENV" value="test"/>
+    <env name="SYMFONY_DEPRECATIONS_HELPER" value="verbose=1"/>
+  </php>
+<!--  <listeners>-->
+<!--    <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>-->
+<!--  </listeners>-->
```

Comment:
Currently is spitting out errors because of incompatibility within the `php-tmdb/api` library itself.